### DISCUSSION TITLE
Demo Mode not working?

### DISCUSSION THREAD

**Discussion by gotopes** [MLB-LED-Scoreboard/mlb-led-scoreboard#513] [General]
Is the demo mode not working?  I just built/installed the scoreboard and I thought I'd go into config.json and set a date for the demo option just to get an idea what it might look like during the season with the games over for the day.  

I get a rotation of today's news headlines and the standings for my demo date of October 1, 2023.

I figured maybe no games were played that day since the season might have ended earlier than I remembered, so i put an api command line in my browser and got a json of games so I know the mlb api is still populated with scores for that day and I am at a loss why it is not in the rotation unless the scores part does not work in demo mode.   

NL scores test: https://statsapi.mlb.com/api/v1/schedule?sportId=1&leagueId=104&hydrate=linescore&date=2023-10-01

Is there also a reason the news and standings repeat twice in a row and not just once before rotating?

Thanks.

**Comment by WardBrian** [comment]
This depends on your config file. The demo mode is not able to simulate the time of day, so all games in the past are considered "finished", e.g., not "live". If you have a config which only shows live games, they won't show up in the demo.

**Comment by gotopes** [comment]
here's a relevant part of my config.json.  Do you see anything obvious in the "true/false" settings keeping the score from showing.  btw, I also see news and standings sections but no section of standings for scores (not even in the config.json.example):

,
        "news_ticker": {
                "team_offday": true,
                "always_display": false,
                "preferred_teams": false,
                "display_no_games_live": true,
                "traderumors": true,
                "mlb_news": false,
                "countdowns": false,
                "date": true,
                "date_format": "%A, %B %-d, %Y"
        },
        "standings": {
                "team_offday": true,
                "mlb_offday": true,
                "always_display": false,
                "display_no_games_live": true
        },
        "rotation": {
                "enabled": true,
                "scroll_until_finished": true,
                "only_preferred": false,
                "only_live": true,
                "rates": {
                        "live": 2.75,
                        "final": 2.75,
                        "pregame": 2.75
                },
                "while_preferred_team_live": {
                        "enabled": false,
                        "during_inning_breaks": false

**Comment by WardBrian** [comment]
Both times it says 
`”display_no_games_live": true, `

**Reply by gotopes** [reply]
ok, i changed those two and only get scores.  i'll do more research on the settings and play around to see if i can get news, standings and scores in rotation.  if not, no biggie until sprig training arrives and i can play around with it more when we have current-day games.

**Reply by WardBrian** [reply]
Rotating between games and the other screens is still a wishlist item for us!
